_episodes/24-diagnosing-issues-improving-robustness.md

```diff
@@ -425,9 +425,9 @@ def patient_normalise(data):
 
     Negative values are rounded to 0.
     """
-    max = np.nanmax(data, axis=1)
+    maxima = np.nanmax(data, axis=1)
     with np.errstate(invalid='ignore', divide='ignore'):
-        normalised = data / max[:, np.newaxis]
+        normalised = data / maxima[:, np.newaxis]
```

Comment:
normalised = data / max[:, np.newaxis]

Reply:
Aha, yes this is what happens when I only look at the immediate vicinity on GitHub. Much easier to spot from my editor! Thanks for catching this.